### DISCUSSION TITLE
更改「当前标签页」「其他标签页」外观状态

### DISCUSSION THREAD

**Discussion by smallg0at** [ywmoyue/biliuwp-lite#123] [General]
4.6.0 更新 WinUI3 时标签样式进行了更新，标签呈现如下的交互状态：

- 当前标签 不高亮
- 悬浮非活动标签 高亮
- 非活动标签 次级高亮

在实际使用中，我发现这种设计不符合 Windows 应用的普遍做法，**我经常会把非活动标签认为是当前标签**，因此希望变更「当前标签」和「其他标签」外观状态：

- 当前标签 高亮
- 悬浮非活动标签 次级高亮
- 非活动标签 不高亮

即类似如下状态（截图自 Edge、Windows 资源管理器，状态顺序和上述文本一致）

<img width="190" alt="image" src="https://github.com/ywmoyue/biliuwp-lite/assets/30719670/060831cb-4760-426b-9754-77ddaddde815">

<img width="509" alt="image" src="https://github.com/ywmoyue/biliuwp-lite/assets/30719670/4f684b56-309e-40d4-9bf4-1748701d13a6">

**Comment by ywmoyue** [comment]
4.6.6已更新标签栏样式，但是我自测在win10上仍有可能不生效，需要手动切换一下深浅色主题
